libbeat/docs/newdashboards.asciidoc

```diff
@@ -18,74 +21,210 @@ export the Kibana dashboards.
 [[import-dashboards]]
 === Importing Existing Beat Dashboards
 
-To import all the dashboards for a Beat, including dependencies such as visualizations, searches, and index
-patterns, you can use the bash script `import_dashboards.sh` (on Unix) or the Powershell script `import_dashboards.ps1`
-(on Windows) from https://github.com/elastic/beats/tree/master/dev-tools[dev-tools]. 
+You can use the `import_dashboards` script to import all the dashboards and the index pattern for a Beat, including the dependencies such as visualizations and searches.
+The `import_dashboards` script is available in the Beat package, under the `scripts` directory.
 
-For example, to import the dashboards for Packetbeat, you can run the following command from the `packetbeat` directory in the `beats` repository:
+You can import the dashboards and the index pattern for a Beat:
 
-*Unix*:
+- from a local directory:
 
 [source,shell]
 ----------------------------------------------------------------------
-../dev-tools/import_dashboards.sh -dir etc/kibana
+$ scripts/import_dashboards -dir etc/kibana
 ----------------------------------------------------------------------
 
-*Windows*:
+- from a local zip archive:
 
 [source,shell]
 ----------------------------------------------------------------------
-..\dev-tools\import_dashboards.ps1 -dir .\etc\kibana
+$ scripts/import_dashboards -beat metricbeat -file beats-dashboards-5.0.0-alpha6.zip
 ----------------------------------------------------------------------
 
+- from a zip archive available online:
+
+[source,shell]
+-----------------------
+$ scripts/import_dashboards -beat metricbeat -url https://download.elastic.co/beats/dashboards/beats-dashboards-5.0.0-alpha6.zip
+-----------------------
+
+You can pass any zip archive with the dashboards for a single Beat or for multiple Beats.
+
+If the zip archive contains dashboards for a single Beat (mybeat-dashboards-1.2.zip), then it must have the
+following structure:
+
+[source,shell]
+-------------------------
+mybeat-dashboards-1.2/
+    dashboards/
+	search/
+	visualization/
+	index-pattern/
+------------------------
+
+If the zip archive contains dashboards for multiple Beats (beats-dashboards-5.0.0-alpha6.zip), then it must have the
```

Comment:
Instead of hard coding this, I'd use beats-dashboards-{stack-version}.zip (same comment applies to wherever we refer to the archive by name).